src/Lykke.blue.Service.ReferralLinks.Services/Bitcoin/BitcoinApiClient.cs

```diff
@@ -0,0 +1,154 @@
+﻿using Lykke.Bitcoin.Api.Client.AutoGenerated;
+using Lykke.Bitcoin.Api.Client.AutoGenerated.Models;
+using Lykke.Bitcoin.Api.Client.BitcoinApi.Models;
+using Lykke.blue.Service.ReferralLinks.Core.Settings.ServiceSettings;
+using System;
+using System.Threading.Tasks;
+
+namespace Lykke.blue.Service.ReferralLinks.Services.Bitcoin
+{
+    public class BitcoinApiClientLocal
```

Comment:
Why do you need this class when you have IBitcoinApiClient?

Reply:
BitcoinApiClientLocal has been removed